packages/typespec-python/generator/pygen/codegen/serializers/builder_serializer.py

```diff
@@ -410,14 +410,19 @@ def _json_response_template_name(self) -> str:
         return "response.json()"
 
     @staticmethod
-    def declare_non_inputtable_constants(builder: RequestBuilderType) -> List[str]:
+    def declare_non_inputtable_headers_queries(builder: RequestBuilderType) -> List[str]:
```

Comment:
It is used for "construct headers and queries" so we had better update the function name:
https://github.com/Azure/autorest.python/blob/3b89887f50d2312786a8e51f6409c993d9143f1b/packages/typespec-python/generator/pygen/codegen/serializers/builder_serializer.py#L469-L498